speakeasy/winenv/api/usermode/kernel32.py

```diff
@@ -5567,3 +5567,158 @@ def AddVectoredExceptionHandler(self, emu, argv, ctx={}):
         emu.add_vectored_exception_handler(First, Handler)
 
         return Handler
+
+    @apihook("GetSystemDefaultUILanguage", argc=0)
+    def GetSystemDefaultUILanguage(self, emu, argv, ctx={}):
+        '''
+        LANGID GetSystemDefaultUILanguage();
+        '''
+        return int("0x0409", 16) #"English (United States)"
+
+    @apihook("GetUserDefaultLangID", argc=0)
+    def GetUserDefaultLangID(self, emu, argv, ctx={}):
+        '''
+        LANGID GetUserDefaultLangID();
+        '''
+        return int("0x0409", 16) #"English (United States)"
+
+    @apihook("FindResourceExW", argc=4)
+    def FindResourceExW(self, emu, argv, ctx={}):
+        '''
+        HRSRC FindResourceExW(
+            [in, optional] HMODULE hModule,
+            [in]           LPCWSTR lpType,
+            [in]           LPCWSTR lpName,
+            [in]           WORD    wLanguage
+        );
+        '''
+        # repeates code in FindResource()
+        cw = self.get_char_width(ctx)
+        hModule, lpType, lpName, wLanguage = argv
+        if hModule == 0:
+            pe = emu.modules[0][0]
+        else:
+            pe = emu.get_mod_from_addr(hModule)
+            if pe and hModule != pe.get_base():
+                return 0
+
+        name = self.normalize_res_identifier(emu, cw, lpName)
+        type_ = self.normalize_res_identifier(emu, cw, lpType)
+        res = self.find_resource(pe, name, type_)
+        if res is None:
+            return 0
+
+        hnd = self.get_handle()
+        self.find_resources.update({hnd: {"ptr": pe.get_base() + res.data.struct.OffsetToData,
+                                          "size": res.data.struct.Size}})
+        return hnd
+
+    @apihook("GetUserDefaultLCID", argc=0)
+    def GetUserDefaultLCID(self, emu, argv, ctx={}):
+        '''
+        LCID GetUserDefaultLCID();
+        '''
+        # https://docs.microsoft.com/en-us/windows/win32/intl/locale-user-default
+        return int("0x0400", 16)
```

Comment:
```suggestion
        return 0x400
```

what does this correspond to?